diff --git a/node_modules/vitest/dist/vendor-node.a7c48fe1.js b/node_modules/vitest/dist/vendor-node.a7c48fe1.js
index 6cb555c..dd4091e 100644
--- a/node_modules/vitest/dist/vendor-node.a7c48fe1.js
+++ b/node_modules/vitest/dist/vendor-node.a7c48fe1.js
@@ -10218,12 +10218,24 @@ class WebdriverBrowserProvider {
       });
     }
     const { remote } = await import('webdriverio');
+
+    const capabilities = {
+      browserName: this.browser,
+    }
+
+    if (this.browser === "chrome") {
+      capabilities["goog:chromeOptions"] = { args: [] };
+      if (options.headless) {
+        capabilities["goog:chromeOptions"].args.push("headless");
+      }
+      if (options.binary) {
+        capabilities["goog:chromeOptions"].binary = options.binary;
+      }
+    }
+
     this.cachedBrowser = await remote({
       logLevel: "error",
-      capabilities: {
-        "browserName": this.browser,
-        "wdio:devtoolsOptions": { headless: options.headless }
-      }
+      capabilities
     });
     return this.cachedBrowser;
   }
